Environment_Engine/Compute/MapToLevel.cs

```diff
@@ -41,16 +41,85 @@ namespace BH.Engine.Environment
 {
     public static partial class Compute
     {
-        [Description("Organize the BHoM Room object into a data structure based on the level of each room.")]
-        [Input("rooms", "A collection of BHoM Rooms to be organized.")]
+        [Description("Organise the BHoM Space objects into a data structure based on the level of each space.")]
+        [Input("spaces", "A collection of BHoM Spaces to be organised.")]
         [Input("levels", "A collection of BHoM Levels.")]
         [Input("decimals", "Provide decimal location to define the degree of tolerance for data matching.")]
-        [MultiOutput(0, "roomsByLevel", "A collection of BHoM Rooms group by levels.")]
+        [MultiOutput(0, "spacesByLevel", "A collection of BHoM Spaces grouped by levels.")]
         [MultiOutput(1, "levelsInUse", "A sublist of the BHoM Levels that have Room.")]
-        public static Output<List<List<Room>>, List<oM.Geometry.SettingOut.Level>> MapToLevel(List<Room> rooms, List<oM.Geometry.SettingOut.Level> levels, int decimals = 6)
+        [MultiOutput(2, "spacesNotMapped", "A collection of BHoM Spaces which did not sit neatly on any of the provided levels")]
```

Comment:
Could we also have a ```unusedLevels``` or ```levelsNotMapped``` output? Certainly could deduce this from the original list, but I think it helps explain what the method is doing.